src/templates/components/footer.hbs

```diff
@@ -1,5 +1 @@
-<footer id="footer" class="site-footer container">
-    <div class="site-author">
```

Comment:
Move address info to here

Reply:
Ill do it! But, if o logo goes in the hero and the address goes in the footer, what are we going to put in the sidebar?

Reply:
Done